frontends/mit-open/src/test-utils/index.tsx

```diff
@@ -44,15 +44,7 @@ const renderRoutesWithProviders = (
   // window.SETTINGS is reset during tests via afterEach hook.
   window.SETTINGS.user = makeUserSettings(options.user)
 
-  const router = createMemoryRouter(
-    [
-      {
-        errorElement: <RethrowError />,
-        children: routes,
-      },
-    ],
-    { initialEntries: [url] },
-  )
+  const router = createMemoryRouter(routes, { initialEntries: [url] })
```

Comment:
`< RethrowError />` is still renderd by `renderWithProviders`, but no longer by `renderTestApp`, which now renders the App's routes normally—the same way the `<App />` component does.

The point of RethrowError was to make errors during testing more visible. The `console.error` in `<ErrorPage />` achieves the same thing for `renderTestApp` (since we fail our unit tests when there are console errors).